_functions_onlyoffice.sh

```diff
@@ -92,6 +94,10 @@ do_start_onlyoffice() {
 
   echo_info "${DEPLOYMENT_ONLYOFFICE_CONTAINER_NAME} container started"
 
+  # add onlyoffice security paramaters to exo server conf
+  local onlyoffice_secret="acceptance%@ooEfqwdededefsadecvd"
```

Comment:
This key should be injected via an environment variable declared on the ci server.
IMO, what you should do is, if onlyoffice is enabled, checking the key is well setted on the environment